shared/middleware/files_api.rb

```diff
@@ -76,11 +92,15 @@ def allowed_file_type?(endpoint, extension)
     not_found if result[:status] == 'NOT_FOUND'
     not_modified if result[:status] == 'NOT_MODIFIED'
     last_modified result[:last_modified]
+
+    abuse_score = result[:metadata]['abuse_score'].to_i
+    not_found if abuse_score > 0 and !can_view_abusive_assets?(encrypted_channel_id)
+
     result[:body]
   end
 
   #
-  # PUT /v3/(assets|sources)/<channel-id>/<filename>?version=<version-id>
+  # PUT /v3/(assets|sources)/<channel-id>/<filename>?version=<version-id>&abuse_score=<abuse_score>
```

Comment:
Remove `abuse_score` from this API, unless there's some place we're setting (or plan to set) an abuse score on an individual asset?